### PR TITLE
Adddlas2

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: testlapack/dlas2.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+type Dlas2er interface {
+	Dlas2(f, g, h float64) (min, max float64)
+}
+
+func Dlas2Test(t *testing.T, impl Dlas2er) {
+	for i, test := range []struct {
+		f, g, h, ssmin, ssmax float64
+	}{
+		{10, 30, 12, 3.567778859365365, 33.634371616111189},
```
Comment (kortschak) [inline testlapack/dlas2.go]: Values from...?

**Comment by kortschak** [comment]
LGTM

File: testlapack/dlas2.go (hunk 1 above)
Comment (btracey) [inline testlapack/dlas2.go]: Octave

Comment (kortschak) [inline testlapack/dlas2.go]: Would you write that in.

**Pull request merged**